src/roles/vsc-backup/vars/main.yml

```diff
@@ -4,6 +4,7 @@ vsc_creds:
   password: "{{ vsc_password|default(vsc_default_password) }}"
   timeout: "{{ vsc_command_timeout_seconds }}"
 vsc_scp_backup_primary_image: "sshpass -p'{{ vsc_password | default(vsc_default_password) }}' scp {% if ssh_proxy_configuration is defined %} -o ProxyJump={{ ssh_proxy_configuration }}{% endif %} -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null {{ vsc_username | default(vsc_default_username) }}@{{ mgmt_ip }}:{{ bof_json['primary_image_unix'] }} {{ backup_folder }}"
+vsc_scp_backup_primary_config: "sshpass -p'{{ vsc_password | default(vsc_default_password) }}' scp {% if ssh_proxy_configuration is defined %} -o ProxyJump={{ ssh_proxy_configuration }}{% endif %} -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null {{ vsc_username | default(vsc_default_username) }}@{{ mgmt_ip }}:/config.cfg {{ backup_folder }}"
```

Comment:
This command is the same as the existing vsc_scp_backup_config command. We need to backup  the primary config file regardless of the name (default is config.cfg, but a custom config file could be named anything - it could also be pointing to a remote location). You can utilize bof_json to get the primary_config, the backup_primary_image command above is an example